Sources/GRPCCodeGen/Internal/Namer.swift

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2024, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package struct Namer: Sendable, Hashable {
+  let grpcCore: String
+
+  package init(grpcCore: String = "GRPCCore") {
+    self.grpcCore = grpcCore
+  }
+
+  private func grpcCore(_ typeName: String) -> ExistingTypeDescription {
+    return .member([self.grpcCore, typeName])
+  }
+
+  private func requestResponse(
+    for type: String?,
+    isRequest: Bool,
+    isStreaming: Bool,
+    isClient: Bool
+  ) -> ExistingTypeDescription {
+    let prefix = isStreaming ? "Streaming" : ""
+    let peer = isClient ? "Client" : "Server"
+    let kind = isRequest ? "Request" : "Response"
+    let baseType = self.grpcCore(prefix + peer + kind)
+
+    if let type = type {
+      return .generic(wrapper: baseType, wrapped: .member(type))
+    } else {
+      return baseType
+    }
+  }
+
+  func literalNamespacedType(_ type: String) -> String {
+    return self.grpcCore + "." + type
+  }
+
+  func serverRequest(forType type: String?, streaming: Bool) -> ExistingTypeDescription {
```

Comment:
Nit: I think having `isStreaming` in all these funcs instead of `streaming` makes it more consistent with `requestResponse` (and it's also more idiomatic).
```suggestion
  func serverRequest(forType type: String?, isStreaming: Bool) -> ExistingTypeDescription {
```

Reply:
Done in https://github.com/grpc/grpc-swift/pull/2201/commits/8fa87bf271d8b7d12e8ebca9143db6e5fe2d7803